src/UIKit/UIEnums.cs

```diff
@@ -300,6 +300,7 @@ public enum UIButtonType : nint {
 		InfoLight,
 		InfoDark,
 		ContactAdd,
+		[TV (11,0)][NoiOS]
```

Comment:
🤔 The `[NoiOS]` is a breaking change, isn't it?

Reply:
`UIButtonTypePlain API_AVAILABLE(tvos(11.0)) __IOS_PROHIBITED __WATCHOS_PROHIBITED` since Xcode 9... Was added here: https://github.com/xamarin/xamarin-macios/commit/35462f0bf75efe2ab877b6c70165d01d25fc5cdb#diff-b632d9f00c73cdcefaee88596ecfbc42R302

RCA: Sharpie does not generate any availability attribute for the enum value, here's the output:

```
[Native]
public enum UIButtonType : nint
{
	Custom = 0,
	System,
	DetailDisclosure,
	InfoLight,
	InfoDark,
	ContactAdd,
	Plain,
	RoundedRect = System
}
```

Sebastien found that by looking at the diff and Miguel probably added the enum straight from Sharpie.

Reply:
No, it's not. That would break the automatic, sequential numbering to the members.
From `UIButtonType.g.cs`
```
		[Introduced (PlatformName.TvOS, 11,0)]
		[Unavailable (PlatformName.iOS, PlatformArchitecture.All)]
		Plain = 6,
```

Reply:
Fair enough :)

Reply:
But I do understand that the `[NoiOS]` behavior is different for enum values (not removed) than members or classes.

Reply:
Bug filed against sharpie: https://bugzilla.xamarin.com/show_bug.cgi?id=60599, to generate the availability attribute for enum values.

Reply:
@VincentDondain the original source does not specify that `Plain = 6` it's implied by its position inside the enum. If it was not compiled in the bindings (defined out) then the generator would not know it must skip a number for the next member. E.g.

`enum Numbers { One, [NoiOS] Two, Three }`
!=
`enum Numbers { One, #if !__IOS__ Two, #endif Three }`